fetch.bs

```diff
@@ -2341,8 +2347,15 @@ be used as part of the cache key.
  <var>origin</var>'s <a for=origin>host</a>.
 </ol>
 
-<p class=note>The same caveat applies. Do not build on this without having a considered discussion
-with the Fetch Standard community first.
+<div class=note>
+ <p>The same caveat applies. Do not build on this without having a considered discussion
+ with the Fetch Standard community first.
+
+ <p>It is <a>implementation-defined</a> whether other DNS queries might be performed to get
+ connection information beyond just <a for=/>IP addresses</a>. For example, if <var>origin</var>'s
+ <a for=origin>scheme</a> is "http" or "https", the implementation might perform a DNS query for
+ HTTPS RRs. [[SVCB]]
```

Comment:
This shouldn't be a note. It seems this should be part of the step that turns a domain into an IP address, no? Although I suppose this does highlight that the "resolve a domain" algorithm maybe ought to take an origin and not a domain.

Reply:
Moved outside the note into the procedure.  And good idea on "resolve a domain" taking the origin instead of domain.  Since there were already two sections here to take origin or domain, and all the origin one did was call the other with the origin's host, and nothing else seems to directly depend on the domain one, I did some trivial merging to give a single "resolve an origin" procedure.